packages/web-components/fast-foundation/src/__test__/helpers.ts

```diff
@@ -0,0 +1,55 @@
+import type { ViewTemplate } from "@microsoft/fast-element";
+import type { Args } from "@storybook/html";
+import qs from "qs";
+
+/**
+ * Returns a formatted URL for a given Storybook fixture.
+ *
+ * @param id - the Storybook fixture ID
+ * @param args - Story args
+ * @returns - the local URL for the Storybook fixture iframe
+ */
+export function fixtureURL(
+    id: string = "debug--blank",
+    args?: Record<string, any>
+): string {
+    const params: Record<string, any> = { id };
+    if (args) {
+        params.args = qs
```

Comment:
I don't know if I should comment here, but I tried something similar in my project. I think you could add a validation for the args in this method as Storybook does not allow specific characters in it's url. [See here](https://storybook.js.org/docs/react/writing-stories/args#setting-args-through-the-url)

This could help some developers who are going to use any of those characters in their arguments as they get a better error response.

Also, I just wanted to thank you for the great work on this PR. Really helped me a lot!

Reply:
Hi @sonntag-philipp, comments and contributions are always welcome!

I dug into the source for Storybook args a few days ago ~~and unfortunately they don't have a function that we could import to validate the args~~ I did some more searching and found the args validator, linked below. They do at least use `qs`, and they also apply some extra formatting in both the encoding and decoding of story args. Our method can definitely be improved to more closely match the expected behavior, which is handled in these files:

* The args query string parser: https://github.com/storybookjs/storybook/blob/v6.5.9/lib/preview-web/src/parseArgsParam.ts
* The story args store validator: https://github.com/storybookjs/storybook/blob/v6.5.9/lib/store/src/args.ts#L74

Reply:
On further reading, it looks like the second link might not be as useful as I thought, since it requires us to pass the `argTypes` to validate that the args are correct.

Reply:
Alright, thanks for the input on this. I think I'll just omit the validation within my own project as it's not going to be public. 

Asking Storybook to expose a validation method would be the most future proof way I think - but you may already thought that too.